tests/call_tests.rs

```diff
@@ -240,3 +240,30 @@ fn mutually_recursive_functions() {
     assert_eq!(args.len(), 1);
     //assert_eq!(args[0], SolutionValue::I32(3))
 }
+
+#[test]
+fn test_pretty_path_llvm_instructions() {
+    let funcname = "nested_caller";
+    init_logging();
+    let proj = get_project();
+    let mut em: ExecutionManager<haybale::backend::DefaultBackend> =
+        symex_function(funcname, &proj, Config::default(), None).unwrap();
+    loop {
+        match em.next() {
+            Some(res) => match res {
+                Ok(_) => {
+                    println!("next() worked");
+                }
+                Err(_) => {}
+            },
+            None => break,
+        }
+        let state = em.state();
+        let path = state.get_path();
+        let len = haybale::get_path_length(path);
+        assert_eq!(
```

Comment:
Also, this test really only checks that `get_path_length()` and `pretty_path_llvm_instructions()` match each other in terms of behavior.  I'd like to additionally check that they match the actual path we expect.  I'd also like the test to explicitly fail if `em.next()` returns `Err`.  The test would look something like this:

```
let mut em = ...
let _ = em.next().expect("Expected a path").unwrap();
assert!(em.next().is_none(), "Expected only one path");
let state = em.state();
let pretty_path = state.pretty_path_llvm_instructions();
assert_eq!(pretty_path, r"
  <expected path here>
  "
);
assert_eq!(get_path_length(state.get_path()), pretty_path.matches("\n").count());
```

Reply:
Done, thanks for the example!